content/rendering-with-batch/batch/job_definition.md

```diff
@@ -4,7 +4,7 @@ date: 2021-09-06T08:51:33Z
 weight: 110
 ---
 
-As a last step to configuring AWS Batch, we will register a job definition that will act as a template when we submit jobs.
+As a last step to configuring AWS Batch, we will register a job definition that includes an AWS Batch retry strategy, that will act as a template when we submit jobs.
```

Comment:
Why only highlighting the retry strategy among all the configuration parameters in the job definition?